console/console-init/ui/src/modules/iot-device-detail/components/GatewayMembership/GatewayMembership.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { Link } from "react-router-dom";
+import { useParams } from "react-router";
+import {
+  Flex,
+  FlexItem,
+  Card,
+  Popover,
+  Button,
+  CardBody,
+  Title,
+  CardTitle
+} from "@patternfly/react-core";
+import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
+import { StyleSheet, css } from "aphrodite";
+
+export interface IGatewayMembershipProps {
+  memberOf?: string[];
+}
+
+const styles = StyleSheet.create({
+  card_body: {
+    marginBottom: "1.25rem"
+  },
+  flex_item_margin: {
+    marginLeft: "0.625rem",
+    paddingLeft: "0"
+  },
+  tooltip_margin: {
+    paddingRight: "3.125rem"
+  }
+});
+
+export const GatewayMembership: React.FC<IGatewayMembershipProps> = ({
+  memberOf
+}) => {
+  const { projectname, namespace } = useParams();
+
+  return (
+    <Card className={css(styles.card_body)}>
+      <CardTitle>
+        <Title
+          className={css(styles.tooltip_margin)}
+          id="gateway-membership-title"
+          headingLevel="h1"
+          size="2xl"
+        >
+          Gateway Group Membership &nbsp;&nbsp;
+          <Popover
+            enableFlip={false}
+            bodyContent={<div>Gateway groups to which the device belongs</div>}
+            aria-label="gateway group membership info popover"
+            closeBtnAriaLabel="close gateway membership info popover"
+          >
+            <Button
+              variant="link"
+              id="gateway-membership-help-button"
+              icon={<OutlinedQuestionCircleIcon />}
+            ></Button>
+          </Popover>
+        </Title>
+      </CardTitle>
+
+      <CardBody className={css(styles.flex_item_margin)}>
+        <Flex>
+          <br />
+          {memberOf &&
```

Comment:
what happens if memberOf is not present?

Reply:
![Screenshot from 2020-08-18 16-41-51](https://user-images.githubusercontent.com/23582438/90506360-c1f78680-e171-11ea-9758-21a59c47a7b8.png)

Reply:
Also the color of tooltip icon doesn't seems accurate to me

Reply:
It's the default color for popover provided by patternfly, I would suggest we stick with this color for consistency and discuss it with the UX team

Reply:
@rkpattnaik780 This is the design color. I would suggest to add custom styling to change the color of icon 
![image](https://user-images.githubusercontent.com/29524461/90510317-6381d680-e178-11ea-87e4-da388be40b33.png)